File: Modul 4/Soal 2/PRAK302-2210817220024-ALDIRA PUTRI SHOLEHA.py
nilai = int(input(""))
if(nilai >= 80) and (nilai <= 100):
    print("A")
elif (nilai >= 70) and (nilai <= 79):
    print("B")
elif (nilai >= 60) and (nilai <= 69):
    print("C")
elif (nilai >= 50) and (nilai <= 59):
    print("D")
elif (nilai >= 0) and (nilai <= 50):
    print("E")
else :
    print("nilai {} = {} ".format(nilai))

File: Modul 4/Soal 2/PRAK302-2210817220024-ALDIRA PUTRI SHOLEHA.c
#include <stdio.h>

int main(){
    int nilai;

    scanf("%d", &nilai);

    if(nilai >= 80 && nilai <= 100)
        printf("A\n");
    else if(nilai > 68 && nilai <= 79)
        printf("B\n");
    else if(nilai > 50 && nilai <= 68)
        printf("C\n");
    else if(nilai > 49 && nilai <= 50)
        printf("D\n");
    else if(nilai > 0 && nilai <= 50)
        printf("E\n");

    main();
    return 0;
}

